zoo-css-challenge/style.css

```diff
@@ -9,6 +10,9 @@ header {
   padding: 20px;
 }
 
+#logo{
+  border: none;
+}
```

Comment:
```suggestion
```
Instead of doing `border:none;`, we could instead delete the CSS below that gives the img a border:

```diff
-img {
-  border: 5px solid #8075ff;
-}
```